src/qoi/qoi.pyx

```diff
@@ -13,6 +14,7 @@ class QOIColorSpace(enum.Enum):
     SRGB = qoi.QOI_SRGB
     LINEAR = qoi.QOI_LINEAR
 
+@cython.internal

```

Comment:
This is not a public class, I think. Users should be prevented from accidentally importing it.

Reply:
It looks like `.write` and `.encode` take this as an (optional) parameter?

Reply:
It has no affect in side the same cython module, just told cython do not do something like ```PyModule_AddObject```

Reply:
Cool, looks like we test for this anyway, so if the pipeline is running (which I think runs tests?) then it should be fine.https://github.com/kodonnell/qoi/blob/eedb9cc5c3ef763b6f0e2a64d9ab19143fe9815b/tests/test_qoi.py#L14